packages/material-ui/package.json

```diff
@@ -67,6 +67,7 @@
     "access": "public"
   },
   "engines": {
-    "node": ">=6.0.0"
+    "node": ">=6.0.0",
+    "yarn": "^1.4.0"
```

Comment:
That's my understanding, but I haven't tested it.

Reply:
@oliviertassinari Confirmed, it will impact:

```
yarn add v1.9.4
info No lockfile found.
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error foo-test-123@1.0.0: The engine "yarn" is incompatible with this module. Expected version "^1.10.0".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

Reply:
That's bad then. I was hoping that we can be more explicit and not just add a note to contributing. 

Is this package really not consumable by node < v6 or was this also just for contributors?

Reply:
@eps1lon We don't transpile for node < v6. So yes, we don't support node < v6.